hudi-common/src/main/java/org/apache/hudi/index/secondary/HoodieSecondaryIndex.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.hudi.secondary.index;
+package org.apache.hudi.index.secondary;
```

Comment:
Is this class a public API that a user can extend?  If not, changing the package name is OK.

Reply:
Not public yet. There is no concrete impl in any released version or master. There is a PR in review - #6677